account_due_list_payment_mode/README.rst

```diff
@@ -27,7 +27,7 @@ to check "Show full accounting features" in user permission.
 Usage
 =====
 
-#. Go to *Invoicing > Adviser > Payments and due list* and you'll see the new
+#. Go to *Invoicing > Adviser > Payments and due list* and you will see the new
```

Comment:
Why changing this? It was correct previously.

Reply:
I know. But I think it's better use formal writing. In English, the use of the apostrophe is more colloquial or casual.

Reply:
Yes, I can agree. As always is for not changing too much diff, but this can be kept indeed.